### PR TITLE
Added JEST for automatic unit testing

### PR DESCRIPTION
**Pull request by torland-klev**
1. Added JEST as the test-runner and assertion library to execute unit tests.
2. Added a new config, _jest.config.ts_, that will set up JEST to run with TypeScript.
3. Added two sub-folders to the src-folder, namely main and test.
4. Wrote unit tests for some methods of lib/fnr, mostly to verify that JEST works.
5. Fixed a bug with dates in February.
6. Implemented validateFNR-function to return which type of FNR it was valid as.

### CODE REVIEW

**Comment by torland-klev** [comment]
1. Added JEST as the test-runner and assertion library to execute unit tests.
2. Added a new config, _jest.config.ts_, that will set up JEST to run with TypeScript.
3. Added two sub-folders to the src-folder, namely main and test.
4. Wrote unit tests for some methods of lib/fnr, mostly to verify that JEST works.
5. Fixed a bug with dates in February.
6. Implemented validateFNR-function to return which type of FNR it was valid as.
7.

File: src/main/lib/fnr/birthdate.ts [hunk 1]
```diff
@@ -27,8 +27,7 @@ export const validateAndCreateBirthday = (
 ): string => {
   if (month === 2) {
     if (day > 28) isLeapYear(year) ? (day = 29) : (day = 28)
-  }
-  if (isDayToBigForMonth(day, month)) day = 30
+  } else if (isDayToBigForMonth(day, month)) day = 30
```
Comment (tanettrimas) [inline src/main/lib/fnr/birthdate.ts:30]: Er du ikke glad i brackets eller

**Review by tanettrimas — APPROVED** [review]

File: src/main/lib/fnr/birthdate.ts (hunk 1 above)
Comment (torland-klev) [reply src/main/lib/fnr/birthdate.ts:30]: Brackets er fryktelige 😵‍💫

Comment (torland-klev) [reply src/main/lib/fnr/birthdate.ts:30]: Neida, jeg skylder på Prettier

**Pull request closed without merging**